### PR TITLE
remote: add delete-pair cli command

### PR DESCRIPTION
**Pull request by loriwitt**

### CODE REVIEW

File: pymobiledevice3/cli/remote.py [hunk 1]
```diff
@@ -165,11 +154,39 @@ def cli_start_tunnel(udid: str, secrets: TextIO, script_mode: bool, max_idle_tim
...
                 debug=True)
 
 
+@remote_cli.command('delete-pair')
+@click.option('--udid', help='UDID for a specific device to delete the pairing record of')
+@sudo_required
+def cli_delete_pair(udid: str):
+    """ delete a pairing record """
+    rsd = select_device(udid)
+    pair_record_path = Path(f'{get_home_folder()}{os.path.sep}remote_{rsd.udid}.plist')
```
Comment (doronz88) [inline pymobiledevice3/cli/remote.py]: this path should be fetched using an external function

File: pymobiledevice3/cli/remote.py [hunk 2]
```diff
@@ -165,11 +154,39 @@ def cli_start_tunnel(udid: str, secrets: TextIO, script_mode: bool, max_idle_tim
...
+    if not verify_tunnel_imports():
+        return
+    rsd = select_device(udid)
     asyncio.run(tunnel_task(rsd, secrets, script_mode, max_idle_timeout=max_idle_timeout, protocol=protocol),
                 debug=True)
 
 
+@remote_cli.command('delete-pair')
+@click.option('--udid', help='UDID for a specific device to delete the pairing record of')
+@sudo_required
```
Comment (doronz88) [inline pymobiledevice3/cli/remote.py:181]: it doesn't require

**Review by doronz88 — CHANGES_REQUESTED** [review]

File: pymobiledevice3/cli/remote.py (hunk 2 above)
Comment (loriwitt) [reply pymobiledevice3/cli/remote.py:181]: It does, in order to delete the pair record created by sudo

File: pymobiledevice3/cli/remote.py (hunk 1 above)
Comment (loriwitt) [reply pymobiledevice3/cli/remote.py]: fixed

**Review by doronz88 — APPROVED** [review]

**Pull request merged**